### DISCUSSION TITLE
Plans for Version 3.2.2

### DISCUSSION THREAD

**Discussion by Tatsu-syo** [Tatsu-syo/noMeiryoUI#92] [Ideas]
I plan to include these features on version 3.2.2.

- add translations when translations are coming.
- add country specific features such as Korean font name translation.
  - display font fallback for Japan.
- Icon spacing adjustment (need research)

Display font fallback in Japan is coping with Japanese Windows system font.
Japanese Windows system fonts are changed on new Windows frequently.
Other country Segoe UI is used since Windows Vista and Segoe UI is stable.

In Japan MS UI Gothic was used in Windows XP. It was bitmap font.
In Windows Vista and 7, Meiryo（メイリオ) was used in system font. This font is suitable for display.
In Windows 8 and 8.1, Meiryo UI  was used in system font. This font is too ugly! And system font change control panel was disappeard. This is why I made this software.
In Windows 10 and 11. Yu Gothic UI is used in system font. This font like outline font  version of MS UI Gothic. It's looks poor than other country.
Later version of Windows 10, BIZ UDPGothic font was added. It looks good and I want to use this font in Japanese version.
Now, Japanese version display font is greatest common divisor selection.
I want to improve looks in Japanese. Because I use in Japanese :).

**Comment by mon-jai** [comment]
Maybe you can consider making the fallback font configurable?

**Reply by Tatsu-syo** [reply]
So, I think add country specific setting in language file, and if language file don't have fallback configuration, use oOS common font like Courier new.
How about?

**Reply by mon-jai** [reply]
> So, I think add country specific setting in language file

It would be better if we could choose the font instead of just applying predefined country-specific settings.

> if language file don't have fallback configuration, use oOS common font like Courier new.

If the user does not specify the fallback font, I think it is better to leave the decision to Windows (just the way we do it now).

Windows does a decent job of picking a fallback font.

**Reply by Tatsu-syo** [reply]
> If the user does not specify the fallback font, I think it is better to leave the decision to Windows (just the way we do it now).
> 
> Windows does a decent job of picking a fallback font.

I remember MS Shell Dlg.

Thanks.

**Comment by Tatsu-syo** [comment]
All functions in next version are completed.
Next version is 3.3.0.

Next version I add these functions.

- Users can choice application's font.
- Adjust application's font in Japanese (in Japan standard font is different from Windows Vista/7, 8.1, 10/11)
- Users can choice running application multiple or not.

I plan to release new version in Feb 25, 2004(JST).

For translators, please see 
[Discussions issue towards next to Version 3.2.1.1](https://github.com/Tatsu-syo/noMeiryoUI/issues/81)
